src/coreclr/vm/arm64/crthelpers.asm

```diff
@@ -1,176 +1,91 @@
 ; Licensed to the .NET Foundation under one or more agreements.
 ; The .NET Foundation licenses this file to you under the MIT license.
 
-;; ==++==
-;;
+; ==++==
+;
 
-;;
-;; ==--==
+;
+; ==--==
+; ***********************************************************************
+; File: CrtHelpers.asm, see history in asmhelpers.asm
```

Comment:
What is this "see history in asmhelpers.asm" comment trying to say? I think this whole comment block should be deleted. You can also delete it in the amd64 files that you have copied it from.

Reply:
My impression was that at some point this functionality was extracted out from `asmhelpers.asm` and added this comment. But looking at the history it is there since the initial commit to coreclr repo, so probably better to just delete them.

Reply:
I actually wonder why the code has to be copied directly from the CRT. Unless the CRT does not automatically link them in statically?

Reply:
> I actually wonder why the code has to be copied directly from the CRT. Unless the CRT does not automatically link them in statically?

Not sure if I understand. @jkotas is referring to the content of `.asm` files (including comments) that I copied from amd64 folder's `crthelpers.asm` and later edited to have arm64 assembly code.